src/core/image.js

```diff
@@ -288,7 +292,7 @@ class PDFImage {
     });
   }
 
-  static createMask({
+  static createMaskForType3({
```

Comment:
Given that this is no longer used only for Type3-fonts, could we perhaps use a slightly more generic name?
Naming things is always hard, but how about e.g. `createFullMask`, `createRawMask`, or some other (better) name?

Also, please remember to update the method name in the `assert` call used in this method.